crates/rbuilder/src/live_builder/mod.rs

```diff
@@ -248,6 +251,11 @@ where
                 "Got header for slot"
             );
 
+            // notify the order pool that there is a new header
+            if let Err(err) = header_sender.send(parent_header.clone()).await {
+                warn!("Failed to send header to builder pool: {:?}", err);
+            }
```

Comment:
are these changes meant for a different branch?

Reply:
Built on top of #314 by mistake.